gnssanalysis/_version.py

```diff
@@ -113,7 +125,11 @@ def run_command(commands, args, cwd=None, verbose=False, hide_stderr=False,
     return stdout, process.returncode
 
 
-def versions_from_parentdir(parentdir_prefix, root, verbose):
+def versions_from_parentdir(
+    parentdir_prefix: str,
+    root: str,
+    verbose: bool,
+) -> Dict[str, Any]:
     """Try to determine the version from the parent directory name.
```

Comment:
One-line sphinx docstring (missing the description of input parameters, errors raised, what's returned, etc.: `:param` `:return`)

Reply:
file autogenerated by the versionner script. so won't be addressed